etc/default/readflow.env

```diff
@@ -4,7 +4,9 @@
 
 # Authentication method
 # - `mock`: Generic user use for testing
-# - `proxy`: Use "X-WEBAUTH-USER" or "X-Auth-Username" HTTP header to extract username
+# - `proxy`: For Basic Authentication Use "X-WEBAUTH-USER" or "X-Auth-Username" HTTP header to extract username
+#            For Authelia "Remote-User" and "Remote-Name" are supported (https://www.authelia.com)
+#            Set 'REACT_APP_AUTHORITY=mock' when bulding UI
```

Comment:
I think you can remove Authelia metion because those headers are also correct for other solutions involving a reverse proxy such as Traefik, NGINX or HAProxy.
```suggestion
# - `proxy`: For proxied authentication that use "X-WEBAUTH-USER", "X-Auth-Username", "Remote-User" or "Remote-Name" HTTP header as username
#            Note: Set 'REACT_APP_AUTHORITY=mock' when bulding UI
```